R/verb-joins.R

```diff
@@ -10,6 +10,7 @@
 #' * `full_join(x, y)`:  `SELECT * FROM x FULL JOIN y ON x.a = y.a`
 #' * `semi_join(x, y)`:  `SELECT * FROM x WHERE EXISTS (SELECT 1 FROM y WHERE x.a = y.a)`
 #' * `anti_join(x, y)`:  `SELECT * FROM x WHERE NOT EXISTS (SELECT 1 FROM y WHERE x.a = y.a)`
+#' * For SQLite right and full joins are simulated using left joins.
```

Comment:
Sorry, I meant in the documentation for the SQLite backend (i.e. at the top of the file where the code lives)

Reply:
ah, that makes sense! I didn't really notice that there is a backend specific documentation...